ibm/resource_ibm_cm_catalog.go

```diff
@@ -0,0 +1,193 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ibm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/platform-services-go-sdk/catalogmanagementv1"
+)
+
+func resourceIBMCmCatalog() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMCmCatalogCreate,
+		Read:     resourceIBMCmCatalogRead,
+		Delete:   resourceIBMCmCatalogDelete,
+		Importer: &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"rev": &schema.Schema{
```

Comment:
Not set in read method..
why is this attribute only computed?
Can be sent as part of request payload according to API Docs